ui/components/institutional/confirm-remove-jwt-modal/confirm-remove-jwt-modal.js

```diff
@@ -9,13 +9,13 @@ import {
   Box,
   Text,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
   Button,
   BUTTON_VARIANT,
   BUTTON_SIZES,
 } from '../../component-library';
+import { ModalContent } from '../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../component-library/modal-header/deprecated';
 import {
   BorderRadius,
   Display,
```

Comment:
Screenshots showing no visual changes to UI

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 3 17 24 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/ad19ea00-bb0c-4105-b0bb-19628fb1a20d">

### After
<img width="1512" alt="Screenshot 2023-12-10 at 3 17 39 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/35a14725-14cb-4ec2-af4e-14d8fa2ef706">